tests/latex/LatexLintBearTest.py

```diff
@@ -1,19 +1,21 @@
-from bears.latex.LatexLintBear import LatexLintBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
+from queue import Queue
 
-good_file = """
-{.}
-{ sometext }\\
-"""
+from bears.latex.LatexLintBear import LatexLintBear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
 
 
-bad_file = """
-{ .}
-{ Sometext \\
-"""
+@generate_skip_decorator(LatexLintBear)
+class LatexLintBearTest(LocalBearTestHelper):
 
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = LatexLintBear(self.section, Queue())
 
-LatexLintBearTest = verify_local_bear(LatexLintBear,
-                                      valid_files=(good_file,),
-                                      invalid_files=(bad_file,))
+    def test_result(self):
+        self.check_validity(self.uut, ['{.}', '{ sometext }'])
+        self.check_invalidity(self.uut, ['{ .}', '{ sometext }'])
+        results = self.check_invalidity(self.uut, ['2018-12-01'])
+        self.assertEqual('Wrong length of dash may have been used.',
```

Comment:
Do we need this? IMO, `check_invalidity(...)` can detect if it is giving any type of error message. So, don't need to use assertEqual here. 
Check second last line [here](http://api.coala.io/en/latest/_modules/coalib/testing/LocalBearTestHelper.html#LocalBearTestHelper.check_invalidity).

Reply:
@sangamcse I wanted to test result, which show that I properly corrected bug from issue (This was mentioned in issue).

Reply:
Oh. But this message is not matching with the `--format` given in `create_arguments(...)`. Am I missing something? 😃

Reply:
Ohk. Got it. NM 😅

Reply:
As this was about a newline marker, which is mostly needed if there are multiple errors/warnings/etc, please add a test where there are *two* messages emitted by `chktex`, and verify both messages are correctly extracted from the output.

Reply:
@jayvdb I have following problem : When I created such file to test as below, then only first error was passed to messages in results. I saw the same behaviour when I run  coala -b LatexLintBear --files=test.tex on actual version available for user. 

```
2018-12-01
Let $t\in[0,\infty)$.
```